os/board/rtl8721csm/src/component/soc/realtek/amebad/fwlib/ram_hp/rtl8721dhp_app_start.c

```diff
@@ -938,6 +938,10 @@ u32 app_mpu_nocache_init(void)
 	mpu_region_config mpu_cfg;
 	u32 mpu_entry = 0;
 
+	/*	Below MPU setting is No cache init
```

Comment:
Can you please use below comment format for better readability:
/*
 *
 */

Reply:
Also, can you please rephrase it to make it more understandable.

Reply:
I have rephrase it and changes the comment format.
Please let me know is there is any problem.